### PR TITLE
internationalization(but only en and ja)

### PR DESCRIPTION
**Pull request by riko122**
add locales for internationalization.

Thanks for providing this tool.
If possible we want to use this in Japanese

### CODE REVIEW

File: locales/ja.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  "controlbar": {
+    "draftname": "ドラフト名",
+    "controls": {
+      "name": "コントロール",
+      "picks": "段数:",
+      "holes": "穴の数:",
+      "tablets": "カード枚数:"
+    },
+    "rulers": {
+      "name": "ルーラー",
```
Comment (jamesba) [inline locales/ja.json:21]: 「定規」かも知れない？

**Review by jamesba — APPROVED** [review]
ありがとうございます！

This all looks good to me. I know a little Japanese, but not enough to do this translation myself, so thank you very much!

I think you should add a line to the copyright notice saying that you wrote the Japanese translation.

I am curious: why not use 定規 as a translation for ruler?

File: locales/ja.json (hunk 1 above)
Comment (riko122) [reply locales/ja.json:21]: Thank you for your comment.

Most Japanese software with ruler(ex: text editor) use ルーラー as ruler.
So I think it's best to use ルーラー as a translation for ruler.

And where should I add a line to the copyright notice?

**Comment by jamesba** [comment]
Hi, @riko122 could you add a line to the copyright notice saying you wrote the Japanese translation? Once that's done we can merge this in.

**Comment by riko122** [comment]
I added to ja.json. How about this?

**Comment by jamesba** [comment]
Looks good to me!

**Comment by riko122** [comment]
Thank you!

**Pull request merged**